docs/src/pages/start/vs-code-configuration.md

```diff
@@ -1,196 +1,90 @@
-- [TODO Highlight](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight)
-- [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur)
-- [Vue Peek](https://marketplace.visualstudio.com/items?itemName=dariofuzinato.vue-peek)
-- [VS Code Icons](https://marketplace.visualstudio.com/items?itemName=robertohuertasm.vscode-icons)
+```json
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode"
+}
+```
 
-To edit the settings use the command `Open Settings JSON` in the Command Palette in the View menu (ctrl+shift+p).
+### TypeScript
 
-```js
+```json
 {
-  "editor.formatOnPaste": true,
-  "editor.formatOnSave": true,
-
-  "editor.codeActionsOnSave": {
-    "source.fixAll": true
-  },
-
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "vetur.format.defaultFormatter.html": "prettyhtml",
-  "vetur.format.defaultFormatter.js": "prettier-eslint",
-
-  "attrsSorter.order": [
-    "is",
-    "v-for",
-    "v-if",
-    "v-else-if",
-    "v-else",
-    "v-show",
-    "v-cloak",
-    "v-once",
-    "v-pre",
-    "id",
-    "ref",
-    "key",
-    "v-slot",
-    "v-slot.+",
-    "#.*",
-    "slot",
-    "v-model",
-    "v-model.+",
-    "v-bind",
-    "v-bind.+",
-    ":.+",
-    "v-text",
-    "v-text.+",
-    "v-html",
-    "v-html.+",
-    "class",
-    "v-on.+",
-    "@.+",
-    "name",
-    "data-.+",
-    "ng-.+",
-    "src",
-    "for",
-    "type",
-    "href",
-    "values",
-    "title",
-    "alt",
-    "role",
-    "aria-.+",
-    "$unknown$"
-  ],
-  "todohighlight.isEnable": true,
-  "todohighlight.include": [
-    "**/*.js",
-    "**/*.jsx",
-    "**/*.ts",
-    "**/*.tsx",
-    "**/*.html",
-    "**/*.php",
-    "**/*.css",
-    "**/*.sass",
-    "**/*.scss",
-    "**/*.vue"
-  ],
-  "workbench.iconTheme": "vscode-icons"
+  "typescript.tsdk": "node_modules/typescript/lib"
 }
 ```
 
 ## Debugging a Quasar project in VS Code
 
+1. First, head to [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) and read it thoroughly.
+2. Then, since Quasar is based on Vue, you will also want to refer to [Vue Cookbook for VSCode debugging](https://vuejs.org/v2/cookbook/debugging-in-vscode.html) for setting up debugging Vue apps.
+
 The best approach is to open that in a browser beside this page so you can review these instructions as you are reading those instructions. And apply the changes to your project as you go.
 
 The first step of the Vue cookbook says it is to enable source maps. Quasar automatically enables source maps for development mode. Here is [a good article](https://blog.scottlogic.com/2017/11/01/webpack-source-map-options-quick-guide.html) that describes the different values for the [webpack devtool setting](https://webpack.js.org/configuration/devtool/) (the one that turns on or off source maps.) Quasar uses _cheap-module-eval-source-map_ by default.
```

Comment:
Actually, "the one that controls the source maps" would probably be more accurate, since it doesn't just turn source maps on or off.

I didn't focus on other existing sections, but we can also update them while we are at it.

Reply:
Oh no, I started editing this document some time ago, and now I noticed that I accidentally reversed some of the stuff.

Reply:
This became a random opportunity to review the debugging instructions. I also updated them, thanks.